# Constants.
uint8 STATE_RELEASED = 0
uint8 STATE_PRESSED = 1

# Fields.
uint8 state

